test-cases/longevity/longevity-all-compressions-3days.yaml

```diff
@@ -0,0 +1,50 @@
+                     "cassandra-stress write cl=QUORUM n=50000000 -schema 'replication(factor=3) compression=DeflateCompressor' -port jmx=6868 -mode cql3 native compression=none    -rate threads=100 -pop seq=1..50000000 -log interval=5",
+                     "cassandra-stress write cl=QUORUM n=50000000 -schema 'replication(factor=3) compression=ZstdCompressor'    -port jmx=6868 -mode cql3 native compression=none    -rate threads=100 -pop seq=1..50000000 -log interval=5" ]
+
+stress_cmd: [ "cassandra-stress mixed cl=QUORUM duration=4200m -schema 'replication(factor=3) compression=LZ4Compressor'       -port jmx=6868 -mode cql3 native compression=lz4     -rate threads=50 -pop seq=1..50000000 -log interval=5",
+              "cassandra-stress mixed cl=QUORUM duration=4200m -schema 'replication(factor=3) compression=SnappyCompressor'    -port jmx=6868 -mode cql3 native compression=snappy  -rate threads=50 -pop seq=1..50000000 -log interval=5",
+              "cassandra-stress mixed cl=QUORUM duration=4200m -schema 'replication(factor=3) compression=DeflateCompressor'   -port jmx=6868 -mode cql3 native compression=none    -rate threads=50 -pop seq=1..50000000 -log interval=5",
+              "cassandra-stress mixed cl=QUORUM duration=4200m -schema 'replication(factor=3) compression=ZstdCompressor'      -port jmx=6868 -mode cql3 native compression=none    -rate threads=50 -pop seq=1..50000000 -log interval=5" ]
+
+stress_read_cmd: [ "cassandra-stress read cl=QUORUM duration=4200m -schema 'replication(factor=3) compression=LZ4Compressor'     -port jmx=6868 -mode cql3 native compression=lz4    -rate threads=20 -pop seq=1..50000000 -log interval=5",
+                   "cassandra-stress read cl=QUORUM duration=4200m -schema 'replication(factor=3) compression=SnappyCompressor'  -port jmx=6868 -mode cql3 native compression=snappy -rate threads=20 -pop seq=1..50000000 -log interval=5",
+                   "cassandra-stress read cl=QUORUM duration=4200m -schema 'replication(factor=3) compression=DeflateCompressor' -port jmx=6868 -mode cql3 native compression=none   -rate threads=20 -pop seq=1..50000000 -log interval=5",
+                   "cassandra-stress read cl=QUORUM duration=4200m -schema 'replication(factor=3) compression=ZstdCompressor'    -port jmx=6868 -mode cql3 native compression=none   -rate threads=20 -pop seq=1..50000000 -log interval=5" ]
+
+run_fullscan: 'random, 120'  # 'ks.cf|random, interval(min)''
+
+round_robin: true
+
+n_db_nodes: 6
+n_loaders: 4
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.4xlarge'
+gce_instance_type_db: 'n1-highmem-16'
+gce_n_local_ssd_disk_db: 6
+
+instance_type_loader: 'c5.4xlarge'
+gce_instance_type_loader: 'c2-standard-16'
+scylla_repo_loader: 'https://s3.amazonaws.com/downloads.scylladb.com/rpm/centos/scylla-4.3.repo'
```

Comment:
Do we want to set it to 4.4 repo?

Reply:
what is it anyway?

Reply:
we use it to install c-s on non-AWS tests and it is defined in `test_default.yaml`, so you should remove it